Marlin/Marlin_main.cpp

```diff
@@ -1019,6 +1035,22 @@ inline void get_serial_commands() {
 
       // If command was e-stop process now
       if (strcmp(command, "M112") == 0) kill(PSTR(MSG_KILLED));
+      #if ENABLED(PROBE_FAIL_PANIC)
+        if (strcmp(command, "G26") == 0) {
+          LCD_MESSAGEPGM(WELCOME_MSG);
+          probe_fail = false;
+        }
+        else if(probe_fail && (strchr(command, 'G') != NULL || strchr(command, 'M') != NULL) && strstr(command, "M105") == NULL) {
+          for(uint8_t j=0; j < MAX_CMD_SIZE; j++)
+            command_queue[cmd_queue_index_r][j] = 0;
+          if(commands_in_queue) {
+            commands_in_queue = 1;
+            cmd_queue_index_r = (cmd_queue_index_r + 1) % BUFSIZE;
+          }
+          serial_count = 0;
+          return;
+        }
+      #endif
```

Comment:
@jbrazio currently we don't call `stop()`. The buffer can still get filled when `IsStopped() == true`. To prevent any host from sending commands into the buffer after probing has failed we throw away *all* commands other than `G26`. That's why we handle `G26` in the parser. `M999 S1` could be used for this purpose as well but since we allocated `G26` before this PR and are afraid of having future conflicts with it.